csrstat/utils.py

```diff
@@ -5,31 +5,18 @@
 #
 # =============================================================================
 
-# ===============
-#   Formatting
-# ===============
-# TODO
-# - Add Type Hinting
-# - Google Docstring format: see example_google_docstring.py
-
-# ===============
-#  Testing
-# ===============
-# - Write tests for functions via PyTest
-
-# =============================================================================
-# =============================================================================
 from __future__ import annotations
+from typing import Any, List, Type
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
 import scipy as sp
 import seaborn as sbn
-from typing import Any, List, Type
-
+import utm
 
 class Point:
```

Comment:
do you think it could be worth deleting this class, and initialising a point process from a dataframe instead?